main.go

```diff
@@ -232,6 +232,15 @@ func (h k8sHelpers) GetNode(cli *client.Client) (*api.Node, error) {
 }
 
 func (h k8sHelpers) AddLabels(n *api.Node, labels Labels) {
+	// Remove nfd node labels that no longer apply
```

Comment:
I would prefer changing `AddLabels` -> `AddorUpdateLabels` and moving the remove labels part to another function as Connor mentioned.